fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/LaoDetailViewModel.java

```diff
@@ -259,6 +261,45 @@ protected void onCleared() {
     disposables.dispose();
   }
 
+  /**
+   * Opens the election and publish opening message
+   * triggers OpenElection event on success or logs appropriate error
+   * @param e election to be opened
+   */
+  public void openElection(Election e){
+
+    Log.d(TAG, "opening election with name : " + e.getName());
+    Lao lao = getCurrentLaoValue();
+    if (lao == null) {
+      Log.d(TAG, LAO_FAILURE_MESSAGE);
+      return;
+    }
+
+    Channel channel = e.getChannel();
+    String laoId = lao.getId();
+
+    //The time will have to be modified on the backend
```

Comment:
I don't think you formatted your code with the plugin.

There should be a space after the `//`.